### PR TITLE
Fix command debug output to join tuples as well.

### PR DESCRIPTION
**Pull request by pvalena**

### CODE REVIEW

File: tmt/utils.py [hunk 1]
```diff
@@ -170,7 +170,7 @@ def _run(self, command, cwd, shell):
 
         # Handle the exit code, return output
         if process.returncode != 0:
-            if isinstance(command, list):
+            if isinstance(command, list) or isinstance(command, tuple):
```
Comment (psss) [inline tmt/utils.py]: Let's use `isinstance(command, (list, tuple))` to make it shorter.

File: tmt/utils.py [hunk 2]
```diff
@@ -185,7 +185,7 @@ def run(self, command, message=None, cwd=None, dry=False, shell=True):
         """
         # Use a generic message if none given, prepare error message
         if not message:
-            line = ' '.join(command) if isinstance(command, list) else command
+            line = ' '.join(command) if isinstance(command, list) or isinstance(command, tuple) else command
```
Comment (psss) [inline tmt/utils.py]: Same as above. Also, please make sure the line <= 79 chars. Thanks.

Comment (pvalena) [reply tmt/utils.py]: OH, sure :) didn't see this one comming :D

**Comment by pvalena** [comment]
```diff
diff --git a/tmt/utils.py b/tmt/utils.py
index 44bf197..4fb8687 100644
--- a/tmt/utils.py
+++ b/tmt/utils.py
@@ -170,7 +170,7 @@ class Common(object):

         # Handle the exit code, return output
         if process.returncode != 0:
-            if isinstance(command, list) or isinstance(command, tuple):
+            if isinstance(command, (list, tuple)):
                 command = ' '.join(command)
             raise subprocess.CalledProcessError(process.returncode, command)
         return stdout, stderr
@@ -185,7 +185,10 @@ class Common(object):
         """
         # Use a generic message if none given, prepare error message
         if not message:
-            line = ' '.join(command) if isinstance(command, list) or isinstance(command, tuple) else command
+            if isinstance(command, (list, tuple)):
+                line = ' '.join(command)
+            else:
+                line = command
             message = f"Run command '{line}'."
         self.debug(message)
         message = "Failed to " + message[0].lower() + message[1:]
```

**Pull request merged**